track/raindrops/raindrops.go

```diff
@@ -238,3 +239,22 @@ func examBytesBuffer(pkg *astrav.Package, r *extypes.Response) {
 		r.AppendImprovement(tpl.StringsBuilder.Format("bytes.Buffer"))
 	}
 }
+
+func examCompareLen(pkg *astrav.Package, r *extypes.Response) {
```

Comment:
I'm flailing around a bit with `astrav` at the moment. Is there a less laborious way of finding whether `len()` is called on a string?

Reply:
Oh, that's excellent, thanks. I'm slowly getting the hang of things, mostly by copying your existing code. I wonder if it wouldn't be a good idea to have a little tutorial for Exalysis contributors, showing how to use `astrav` to find a few different types of code issues?

Reply:
astrav is still very much WIP. All it does is putting a simpler layer on top of the ast packages of Go.
The key functionality it provides is moving around in a parent/child manner and finding things in a parent. Most of the functionality can be seen the [Node](https://github.com/tehsphinx/astrav/blob/master/node.go#L41) interface. Access to special node type functionality and the underlying ast nodes methods and values can be gained by casting the node to it's actual astrav type.

That's it basically.

Reply:
Note: my mac broke... so currently I'm kind of cut off. 😢

Reply:
Oh dear! That's bad!